cas-server-support-mongo/src/test/resources/cas.properties

```diff
@@ -0,0 +1,6 @@
+cas.authn.mongo.collection.name=users
+cas.authn.mongo.db.host=mongodb://casuser:Mellon@ds061954.mongolab.com:61954/jasigcas
```

Comment:
To test MongoDB in pac4j, I use an embedded MongoDB: it's a bit heavy (it needs to be downloaded first), but it works well: https://github.com/pac4j/pac4j/blob/master/pac4j-mongo/pom.xml#L57